scripts/benchmark_cortex.py

```diff
@@ -201,113 +235,131 @@ def reportdata(self):
                                 'count': count}))
         return retn
 
+    async def _waitUntilLocked(self, prox: s_telepath.Proxy, layeriden: str):
+        while True:
+            stat = await prox.dyncall(layeriden, s_common.todo('stat'))
+            if stat['lock_goal'] == stat['lock_progress']:
+                return
+            await asyncio.sleep(0.1)
```

Comment:
Maybe we should make a "the layer is completely ready" event?

Reply:
It is technically ready initially, just slow. Adding it would only be useful for benchmarking.